lib/grape-swagger/doc_methods.rb

```diff
@@ -47,6 +47,12 @@ def parse_params(params, path, method)
                           'double'
                         when 'Symbol'
                           'string'
+                        when /^\[(?<type>.*)\]$/
+                          items[:type] = Regexp.last_match[:type].downcase
+                          if PRIMITIVE_MAPPINGS.key?(items[:type])
+                            items[:type], items[:format] = PRIMITIVE_MAPPINGS[items[:type]]
```

Comment:
I re-read this and find it a bit strange (or at least looks like a side-effect) that we're assigning `items[]` inside the case statement. Feels like this logic belongs below where we deal with `items`? I am not sure though, what do you think?